### PR TITLE
Sync Query Params with SearchBar

### PR DESCRIPTION
**Pull request by chalkedgoose**
## Describe your changes
Query parameters persist after initial search and subsequent searches.

Screen Recording 2022-11-19 at 11 04 24 AM mov

## Issue ticket number or describe the feature
https://github.com/charlescohanlon/ProfesScore/issues/9

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [ ] If it is a new feature, I have thoroughly tested it myself

### CODE REVIEW

File: lefthook.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ pre-commit:
   commands:
     eslint:
       glob: "*.{js,ts,jsx,tsx}"
-      run: yarn lint {staged_files}
+      run: pnpm run lint --dir {staged_files}
```
Comment (chalkedgoose) [inline lefthook.yml:6]: There was same strange issue with the pages directory so I added the ```--dir``` arg. 
Also changed it from using yarn to pnpm so it runs with the proper build too.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# lockfile from npm 
+package-lock.json
```
Comment (chalkedgoose) [inline .gitignore:4]: Also an unrelated change deleted the lockfile and added it to the ignore
file since we use pnpm.

File: lefthook.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 pre-commit:
   parallel: true
   commands:
-    eslint:
-      glob: "*.{js,ts,jsx,tsx}"
-      run: yarn lint {staged_files}
+    lintjavascript:
+      glob: "*.{js,ts,tsx}"
```
Comment (chalkedgoose) [inline lefthook.yml:5]: This fixes a problem with lefthook where it could 
not find the pages directory.

**Review by charlescohanlon — APPROVED** [review]
lgtm
